### ISSUE TITLE
Cannot build on Windows 10 with VS Code against EPICS V 7.0.4.1

### ISSUE THREAD

**Issue by whs92** [epics-modules/sscan#22]
I am trying to build sscan on a windows 10 machine using VS Code Compilers. I followed instructions here https://docs.epics-controls.org/projects/how-tos/en/latest/getting-started/installation-windows-plain.html to install:

- EPICS base 7.0.4.1
- asyn
- autosave
- alive
- seq

If I try to build sscan I get the following error: 

```
make[3]: *** No rule to make target 'scanProg.obj', needed by 'scanProgress.dll'.  Stop.
make[3]: Leaving directory 'C:/Users/sissy_user/epics/support/sscan/2-11-4/sscanApp/src/O.windows-x64'
make[2]: *** [C:\Users\sissy_user\epics\base-7.0.4.1\base-R7.0.4.1/configure/RULES_ARCHS:58: install.windows-x64] Error 2
make[2]: Leaving directory 'C:/Users/sissy_user/epics/support/sscan/2-11-4/sscanApp/src'
make[1]: *** [C:\Users\sissy_user\epics\base-7.0.4.1\base-R7.0.4.1/configure/RULES_DIRS:85: src.install] Error 2
make[1]: Leaving directory 'C:/Users/sissy_user/epics/support/sscan/2-11-4/sscanApp'
make: *** [C:\Users\sissy_user\epics\base-7.0.4.1\base-R7.0.4.1/configure/RULES_DIRS:85: sscanApp.install] Error 2
```

[This thread on techtalk](https://epics.anl.gov/tech-talk/2016/msg01446.php) made me think it was probably to do with not getting somthing from the build of the sequencer support module. I have checked that the build succeeded for seq, and done a `make clean uninstall` before typing `make` to ensure that any changes to configure/RELEASE were picked up. 

Any ideas? let me know if I can provide any more information to help diagnose the problem

**Comment by MarkRivers** [comment]
Is sscan/configure/RELEASE pointing to the correct location for SNCSEQ?

**Comment by whs92** [comment]
yes

**Comment by whs92** [comment]
I tried building against base 3.15.7, and that worked.

**Comment by whs92** [comment]
I built https://github.com/FreddieAkeroyd/EPICS-re2c against Base7.0.4.1, then built seq 2.2.8 against that and configured configure/CONFIG_SITE to point to the executable of re2c. Then I was able to build sscan against this version of seq. 

It seemed that without the re2c executable seq will build against base 7.0.4.1, but not 3.15.7. But if it does build, it can't be used by sscan. 

I hope this is useful to somebody else. I will close the issue.
